knowledge_graph/common_relations.py

```diff
@@ -30,7 +30,6 @@
 EVENT_DATA = "event/data"
 EVENT_TYPE = "event/type"
 ACTION_TEXT_VALIDATION = "action/text_validation"
-CONCEPT_TRIGGER = "concept/trigger"
```

Comment:
on purpose?

Reply:
yes, it already exists under a different name.